11ty/definitions/-misia.md

```diff
@@ -0,0 +1,44 @@
+---
+title: -misia
+slug: -misia
+speech: noun
+defined: true
+sub_terms:
+  - text: Fat
+    full_title: fatmisia
+  - text: Islam
+    full_title: fatmisia
+  - text: Trans
+  full_title: transmisia
+reading:
+  - text: Beatley Library Anti-Oppression Guide
+    href: https://simmons.libguides.com/anti-oppression#s-lib-ctab-10174165-1
+  - text: 'Anti-Oppression: Anti-Fatmisia'
+    href: https://simmons.libguides.com/anti-oppression/anti-fatmisia
+---
+---
+from Greek for hate or hatred
+
+## Use
+
+-Misia can be appended to minoritised identifiers that are targeted for hate, such as to fat (fatmisia), trans (transmisia), or Islam (Islamomisia).
+
+## Issues
+
+-Phobia is Greek for 'fear of'.
+
+When used as a suffix in the context of Islamophobia, transphobia, or fatphobia, it is translated to mean that the group has a fear of individuals and communities who identify in those ways. However, the implication and subtext typically is that there is prejudice and discrimination against those groups. It falsely masks hate as fear.
```

Comment:
Some text change suggestions: 

“it is translated to mean” —> use “understood” or “commonly thought” instead?

“mean that the group” —> perhaps go with “group or individual” (I have definitely seen one person use this kind of argument). And then the next bit (“fear of individuals and communities”) could be “people and communities” to avoid repetition. 

For the sentence starting with “However”, what about something like this: “Rather than a direct translation however, the implication and subtext of these terms is one of prejudice and discrimination.”

For the last sentence, use phobia instead of it, just to be super clear: “Using the term phobia falsely masks hate as fear.”